MonoGame.Framework.Content.Pipeline/Processors/TextureProcessorOutputFormat.cs

```diff
@@ -2,6 +2,8 @@
 // This file is subject to the terms and conditions defined in
 // file 'LICENSE.txt', which is part of this source code package.
 
+using System;
+
 namespace Microsoft.Xna.Framework.Content.Pipeline.Processors
 {
     /// <summary>
```

Comment:
Maybe the comment for the `Compressed` case below should be updated?

```
 /// <summary>
        /// The SurfaceFormat value, of the input TextureContent object, is converted to an appropriate compressed format for the target platform.
        /// This can include PVRTC for iOS, DXT for desktop, Windows 8 and Windows Phone 8, and ETC1 or BGRA4444 for Android.
        /// </summary>
```